src/mutations/collection/deleteCollections.ts

```diff
@@ -0,0 +1,39 @@
+import { ForbiddenError, UserInputError } from 'common/errors'
+import { fromGlobalId } from 'common/utils'
+import { MutationToDeleteCollectionsResolver } from 'definitions'
+
+const resolver: MutationToDeleteCollectionsResolver = async (
+  _,
+  { input: { ids } },
+  { dataSources: { collectionService }, viewer }
+) => {
+  if (!viewer.id) {
+    throw new ForbiddenError('Viewer has no permission')
+  }
+  if (ids.length === 0) {
+    return false
+  }
+
+  const unpacked = ids.map((id) => fromGlobalId(id))
+  const types = unpacked.map((d) => d.type)
+
+  if (types.some((type) => type !== 'Collection')) {
```

Comment:
Just FYI, there is a [ticket](https://github.com/thematters/matters-server/issues/1959) about the ID validation, could move to upcoming sprints if needed.